.github/actions/setup-haskell/action.yaml

```diff
@@ -23,6 +23,13 @@ inputs:
     required: false
     default: "waspc"
 
+  extra-cache-key:
```

Comment:
Makes sense! Me with naming again, but what confused me a bit is that this is not the key though, this is only part of the key. So maybe let's rename it to `extra-cache-key-segment` or `unique-cache-key-segment` or something like that?